addonscript/requirements/base.py37.txt

```diff
@@ -180,9 +180,9 @@ jsonschema==3.2.0 \
     --hash=sha256:4e5b3cf8216f577bee9ce139cbe72eca3ea4f292ec60928ff24758ce626cd163 \
     --hash=sha256:c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a
     # via scriptworker
-jwcrypto==0.9.1 \
-    --hash=sha256:12976a09895ec0076ce17c49ab7be64d6e63bcd7fd9a773e3fedf8011537a5f6 \
-    --hash=sha256:63531529218ba9869e14ef8c9e7b516865ede3facf9b0ef3d3ba68014da211f9
+jwcrypto==1.0 \
```

Comment:
Post release bump to 1.0.dev1
    Create SECURITY.md
    Show Code Scan button and exclude trivial pulls
    Allow empty payloads in JWS tokens
    Add tests to check empty payload support
    Drop python2 compatibility
    Fix python3 pylint issues
    Add explicit support to check 'typ' in JWT
    Drop support for importing old MutableMapping
    Disable annoying pep8 naming checks
    Version 1.0
https://github.com/latchset/jwcrypto/releases/tag/v1.0.0